handbook/company/pricing-features-table.yml

```diff
@@ -1116,6 +1116,16 @@
   jamfProtectHasFeature: no
   waysToUse:
     - description: Automatically set admin access to Fleet based on your IDP
+- industryName: Map IdP users to devices (SCIM)
```

Comment:
I think let's try putting this new row below the existing "Human-endpoint mapping":

![Screenshot 2025-03-11 at 4 24 48 PM](https://github.com/user-attachments/assets/c23db1ac-aaf3-4d57-8e92-c8c747cb3644)

Maybe we call them this?
- "Human-endpoint mapping"
- "Human-endpoint mapping (IdP groups)"

That way, users shopping around will realize they can do mapping for free but they have to pay to get IdP groups mapped.